### PR TITLE
Add alt text to images without any

### PR DESCRIPTION
**Pull request by JosephCarpenter**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,7 +27,7 @@ <h3>We are one of the most successful and inventive FIRST Robotics teams in the
 </div>
 <main>
     <div>
-        <img src="/assets/img/worlds2018.jpg">
+        <img src="/assets/img/worlds2018.jpg" alt="The team at the 2018 FIRST World Championships, in front of the FIRST Logo at the Cobo Center in Detroit.">
```
[the remark this answers is not in the document]
Comment (JosephCarpenter) [reply index.html]: oof, there goes 15 minutes of copy pasting and alt texting

[the remark this answers is not in the document]
Comment (ErikBoesen) [reply index.html]: Sorry man. Should have explained this earlier :(

**Comment by ErikBoesen** [comment]
Also heh, #254

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (JosephCarpenter) [reply index.html]: no problem

**Pull request merged**